src/fparser/common/tests/test_sourceinfo.py

```diff
@@ -352,6 +353,21 @@ def test_get_source_info_file(extension, header, content):
             assert source_info == content[1]
 
 
+def test_get_source_info_utf8():
+    encoding = dict(encoding='UTF-8') if six.PY3 else {}
+    with tempfile.NamedTemporaryFile(mode='w', **encoding) as tmp_file:
+        content = u'''
+            ! A fortran comment with a unicode character "{}"
```

Comment:
Could you please use `{0}` rather than `{}`. I know it's not required but it is the approach we've been taking.

Reply:
Done.